app/assets/stylesheets/maps.scss

```diff
@@ -29,6 +31,31 @@
   margin: 10px auto;
 }
 
+<<<<<<< HEAD
```

Comment:
Looks like there is a merge conflict line in here still, probably not causing much of a problem in the css, but can you delete it real quick.